doc/classes/ShaderGlobalsOverride.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ShaderGlobalsOverride" inherits="Node" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Overrides global shader parameters' values in a specific scene.
+		A node used to override shader parameters' values in a scene.
```

Comment:
We could link the shading language page as a tutorial, it has a section on global uniforms:
https://docs.godotengine.org/en/stable/tutorials/shaders/shader_reference/shading_language.html#global-uniforms